download/index.js

```diff
@@ -0,0 +1,99 @@
+var yeoman = require( 'yeoman-generator' );
+var https = require( 'https' );
+var fs = require('fs.extra');
+var chalk = require( 'chalk' );
+var rimraf = require( 'rimraf' );
+
+var MoxieDownload = yeoman.generators.Base.extend( {
+    initializing: function () {
+        this.pkg = require( '../package.json' );
+    },
+    prompting: function(){
+      var done = this.async(); var prompt = {
+        name: 'install',
+        message: 'Do you want to install Wordpress [yes / no]?:'
+      };
+
+      this.prompt(prompt, function(answers){
+        var answer = answers.install.toLowerCase();
+        if( answer === 'yes' ){
+          done();
+        } else {
+          console.log('\n' + chalk.green('Have a nice day!'));
+        }
+      }.bind( this));
+    },
+    createTmpDirectory: function(){
+      var directory = './tmp';
+      console.log('\n' + chalk.bold.yellow('Creating temporarily directory'));
+
+      if(! fs.existsSync( directory ) ) {
+        fs.mkdirSync( directory );
+      } else {
+        console.log( chalk.yellow('Great, we already have one.') );
+      }
+    },
+    downloadWordpress: function(){
+      var url = 'https://wordpress.org/latest.zip'
+      var directory = './tmp';
+      var cb = this.async();
+      var message = 'Downloading & extracting WP in '+  directory + ' directory';
+
+      console.log('\n' + chalk.bold.yellow( message ));
+      this.extract(url, directory, cb);
+    },
+    removeDefaultWPContent: function(){
```

Comment:
Well basically because of the remove of unused themes and plugins in the initial download and to do not have conflicts when the wp-content/themes form the latest version of WP is copied in the current directory where exists a wp-content/themes/names-themegenerated .